Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_sdio_slot0.cpp

```diff
@@ -63,19 +63,28 @@ void CWII_IPC_HLE_Device_sdio_slot0::OpenInternal()
 {
 	const std::string filename = File::GetUserPath(D_WIIROOT_IDX) + "/sd.raw";
 	m_Card.Open(filename, "r+b");
+	if (m_Card)
+		return;
+	
+	WARN_LOG(WII_IPC_SD, "Failed to open SD Card image, trying read-only access...");
+	m_Card.Open(filename, "rb");
+	if (m_Card)
+	{
+		SConfig::GetInstance().bEnableMemcardSdWriting = false;
```

Comment:
If we want proper Wii-like support, what we lack is a way to tell the PPC software that the SD card is read-only, and a way to change SD cards while running. Only being able to change whether a card is read-only when it's not inserted matches the Wii the best. Anyway, just having the flag is good enough for this PR.